### ISSUE TITLE
Issue with climbing power

### ISSUE THREAD

**Issue by LurkingFox6539** [apace100/origins-fabric#305]
There seems to be an issue with the climbing power not working, I can't climb up anything as an arachnid and when I added to power to a custom origin using a datapack it also didn't work.

**Comment by SpinnyBoiii** [comment]
its a toggle-able skill that starts toggled, have you tried pressing your primary skill key?

**Comment by LurkingFox6539** [comment]
I have, it doesn't seem to work even after pressing the primary power key. I even tried it with the secondary power key and still nothing. I might be missing something on how to activate it but if so I'm not sure what.

**Comment by SpinnyBoiii** [comment]
oh i just noticed it got editted with the latest (2?) version, i was still using the older version of it.
based on what i see at the source, ima guess it has to do
"power": "*:*_toggle"
which previously was 
"power": "origins:climbing_toggle"

**Comment by SpinnyBoiii** [comment]
well it works for me when replacing it so problem found i guess

**Comment by miraifrommars** [comment]
My climbing was working until I died for the fifth time overall in the world. Every other aspect of the mod still worked, but for some reason climbing just didn't work. I tried restarting Minecraft and my computer, but nothing helped. However, it turned I had just accidentally pressed G in that time, so I just pressed it again and the climbing works now! 

But also check that you have the right version of fabric and origins etc..
